api.planx.uk/modules/flows/updateTemplatedFlow/controller.ts

```diff
@@ -28,8 +30,11 @@ export const updateTemplatedFlowController: UpdateTemplatedFlowController =
       res.locals.parsedReq.body.payload;
 
     try {
+      const response = await updateTemplatedFlow(sourceFlowId, templatedFlowId);
+
       res.status(200).send({
-        message: `Successfully queued up event to update ${templatedFlowId}`,
+        message: `Successfully updated templated flow on source publish (source ID ${sourceFlowId}, templated flow ID ${templatedFlowId})`,
+        data: response,
```

Comment:
TODO - we don't actually need to pass `summary` into this controller anymore, can easily include in later fetch in `service.ts`